ddtrace/internal/flare/flare.py

```diff
@@ -36,14 +35,28 @@ class FlareSendRequest:
     source: str = "tracer_python"
 
 
+class TracerFlareSendError(Exception):
+    def __init__(self, message: str):
+        super().__init__(message)
+        self.message = message
```

Comment:
Not necessary, but none of the default error types seemed to really fit the failure type here. If there's a generic type that works better, I'm more than happy to clean this up

Reply:
It looks to me like the entire `__init__` you've got defined is redundant. I'm pretty sure this code would work the same if lines 39-41 were replaced with `pass`.